### ISSUE TITLE
PT Feeder not writing All_max_cost_percentage

### ISSUE THREAD

**Issue by b0nadrag** [mehtadone/PTFeeder#296]
If "UseMaxCostPercentage": "true" is set in hostsettings, the ALL_max_cost setting in PAIRS property is filled with the percentage value, rather than ALL_max_cost_percentage (which is absent from the PAIRS file).

My setup:

hostsettings:
    "UseMaxCostPercentage": "true", // whether cost should be as a percentage

appsettings:
    "MaxCost": "2.5", // ALL_max_cost or or All_max_cost_percentage

PAIRS.properties:
#------- Cost / Balance ------
ALL_max_cost = 2.5
ALL_min_buy_balance = 0
ALL_min_buy_balance_percentage = 10

Should the PAIRS.properties file not read:

ALL_max_cost = 0
ALL_max_cost_percentage = 2.5

**Comment by mehtadone** [comment]
Yes it should do. I'm assuming you restarted. I've not experienced some report that before so I'm assuming it's a syntax thing but you never know
